__tests__/__integration__/imperative/__tests__/__integration__/cli/config/__resources__/profiles_with_v1_names/v1profile/myv1profile.yaml

```diff
@@ -0,0 +1,4 @@
+hostname: 'convert hostname to host'
+username: 'convert username to user'
+pass: 'convert pass to password'
+token: 'convert token to takenValue'
```

Comment:
Are we sure that converting `token` to `tokenValue` is always desirable?

For some plugins like Endevor Bridge for Git, "token" is used for a different purpose than connecting to the mainframe. Thanks to @awharn for pointing this out 🙂

According to the web help for the [`zowe endevor-bridge-for-git endevor-connection create`](https://docs.zowe.org/stable/web_help/index.html?p=zowe_endevor-bridge-for-git_endevor-connection_create) command:
![image](https://user-images.githubusercontent.com/22344007/153225756-95a54d93-e645-440f-8ae9-1ec9f8d1422b.png)

Reply:
The conversion of token came from this item in the story:

    Info from Bridge for Git
     - token -> tokenValue should be included
 
We can revisit this question in a standup meeting to confirm if this is the desired action.